examples/cur/reportdefinition.yaml

```diff
@@ -0,0 +1,183 @@
+    forceDestroy: true
+    region: us-east-1
+---
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: BucketPolicy
+metadata:
+  name: example
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: example_cur_report_definition
+spec:
+  forProvider:
+    region: us-east-1
+    bucketSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    policy: |
+      {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "billingreports.amazonaws.com"
+                },
+                "Action": [
+                    "s3:GetBucketAcl",
+                    "s3:GetBucketPolicy"
+                ],
+                "Resource":"arn:aws:s3:::test-definition-bucket-axbjhdwjdk",
+                "Condition": {
+                    "StringEquals": {
+                        "aws:SourceArn": "arn:aws:cur:us-east-1:${data.aws_account_id}:definition/*",
+                        "aws:SourceAccount": "${data.aws_account_id}"
+                    }
+                }
+            },
+            {
+                "Sid": "Stmt1335892526596",
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "billingreports.amazonaws.com"
+                },
+                "Action": "s3:PutObject",
+                "Resource": "arn:aws:s3:::test-definition-bucket-axbjhdwjdk/*",
+                "Condition": {
+                    "StringEquals": {
+                        "aws:SourceArn": "arn:aws:cur:us-east-1:${data.aws_account_id}:definition/*",
+                        "aws:SourceAccount": "${data.aws_account_id}"
+                    }
+                }
+            }
+          ]
+        }
+---
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Role
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: example
+  name: test-role
+spec:
+  forProvider:
+    assumeRolePolicy: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Action": "sts:AssumeRole",
+            "Principal": {
+              "Service": "billingreports.amazonaws.com"
+            },
+            "Effect": "Allow",
+            "Sid": ""
+          }
+        ]
+      }
+---
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Policy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: p
+  name: test-recorder-policy
+spec:
+  forProvider:
+    policy: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "s3:GetBucketAcl",
+              "s3:GetBucketPolicy"
+            ],
+            "Resource":"arn:aws:s3:::test-definition-bucket-*",
```

Comment:
Should we revert wildcards here (and a couple of more below) as well?

Reply:
It's up to you. Unlike the BucketPolicy, in the Policy wildcards work well.